fmriprep/workflows/bold/base.py

```diff
@@ -647,6 +658,22 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
             (bold_surf_wf, outputnode, [('outputnode.surfaces', 'surfaces')]),
         ])
 
+        # CIFTI output
+        if cifti_output and 'template' in output_spaces:
+            cifti = pe.MapNode(GenerateCifti(), iterfield=["surface_target", "gifti_files"],
+                               name="gen_cifti")
```

Comment:
Node "name" fields should match the node variable names. Makes it easier to associate code with locations in the working directory. `gen_cifti` seems more descriptive, so I'd go with that for the variable name.